e2e/credential-revocation.e2e.ts

```diff
@@ -277,13 +294,21 @@ describe('Off-chain credential revocation', () => {
 
     const verifiableCredential = extractCredential(claim);
 
-    mockCredentialInitiate(issuer.signerService.didHex);
+    mockCredentialInitiate(revoker.signerService.didHex);
```

Comment:
Could this be a bit more accurate? It is initiating the revocation?
```suggestion
    mockCredentialInitiateRevocation(revoker.signerService.didHex);
```